astroquery/utils/commons.py

```diff
@@ -467,3 +468,89 @@ def parse_votable(content):
     """
     tables = votable.parse(BytesIO(content), verify='warn')
     return tables
+
+
+def static_vars(**kwargs):
+    def decorate(func):
+        for k in kwargs:
+            setattr(func, k, kwargs[k])
+        return func
+    return decorate
+
+
+@static_vars(caps={})
```

Comment:
Wouldn't it be simpler to make `caps` a module level `dict` instead of a function attribute?

Reply:
It would, but then the `commons` would have a field member that would only apply to a single function which seems like a waste.  Is `commons` meant to be used that way?